facia-tool/public/css/style.css

```diff
@@ -914,6 +914,10 @@ collection-widget:first-child .collection {
     color: #CCCCCC;
 }
 
+.article__ammends .label--premium {
```

Comment:
Does this need to be so specific? Good practise is to look at the 'label' as an object on its own. It should work without knowing what its parent is.

In this case '.label--premium' is tightly coupled with '.article__ammends' which means that you can't use this object anywhere else on the page unless it's nested under '.article__ammends'.